cmd/external-plugins/automated-approver/README.md

```diff
@@ -0,0 +1,61 @@
+# Automated Approver
+
+## Overview
+
+With the Automated Approver tool, you can automatically approve pull requests based on the rules you define. The tool enables automation of the approval process for pull requests in repositories that need reviews before merge. The tool automates the PR review process without limiting user write permission on the repository. It can provide an automated review process for all pull request authors.
+
+## How it works
+
+```mermaid
+flowchart TB
+    A[Github Event]-->B[Prow]
+    B-->|Dispatch event to plugins|C[Automated Approver]
+    C-->D{"`Does PR
+    meet rules?`"}
+    D-->|NO|Z[End]
+    
+    D-->|Yes|E{"`Does PR tests
+    finished?`"}
+    E-->|No|F{"`Did hit timeout?`"}
+    F-->|No|G[Sleep, waiting for tests]
+    G-->E
+    F-->|Yes|Z
+    
+    E-->|Yes|H{"`Does PR statusses
+    finished with success?`"}
+    H-->|No|Z
+    H-->|Yes|I[Approve PR]
+```
```

Comment:
```suggestion

```mermaid
flowchart TB
    A[Github Event]-->B[Prow]
    B-->|dispatch event to plugins|C[Automated Approver]
    C-->D{"`PR adheres
    to rules?`"}
    D-->|NO|Z[End]
    
    D-->|Yes|E{"`PR tests
    completed?`"}
    E-->|No|F{"`hit timeout limit?`"}
    F-->|No|G[sleep, wait for tests]
    G-->E
    F-->|Yes|Z
    
    E-->|Yes|H{"`PR statuses
    finished with success?`"}
    H-->|No|Z
    H-->|Yes|I[Approve PR]
```
```

Reply:
Replaced with .svg file.